### PR TITLE
add build-in topics page

### PR DESCRIPTION
**Pull request by wenxijuji**
This is to add the build-in topics page

### CODE REVIEW

File: docs/topics.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# Juji's Build-in Library Topics
```
Comment (huahaiy) [inline docs/topics.md]: Built-in

File: docs/topics.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+## Fallback topics
+Most of the build-in fallback topics do not expect any argument.
```
Comment (huahaiy) [inline docs/topics.md]: built-in

Comment (huahaiy) [reply docs/topics.md]: May explain a bit what is "Fallback topics"

**Comment by wenxijuji** [comment]
@huahaiy Updated the PR according to the comments

**Pull request merged**